src/main/java/com/textmessenger/config/ReactResourceResolver.java

```diff
@@ -0,0 +1,55 @@
+package com.textmessenger.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.web.servlet.resource.ResourceResolver;
+import org.springframework.web.servlet.resource.ResourceResolverChain;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+public class ReactResourceResolver implements ResourceResolver {
+
+  private static final String REACT_DIR = "/static/";
+  private static final String REACT_STATIC_DIR = "static";
+  private Resource index = new ClassPathResource(REACT_DIR + "index.html");
+  private List<String> rootStaticFiles = Arrays.asList("favicon.ico",
+          "asset-manifest.json", "manifest.json", "service-worker.js");
+
+  private Resource resolve(String requestPath, List<? extends Resource> locations) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "locations". ([squid:S1172](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=200&issues=AWYAbmqdM63is4RDlAg_&open=AWYAbmqdM63is4RDlAg_)

[ISSUE_KEY]: AWYAbmqdM63is4RDlAg_